netweaver/install_aas.sls

```diff
@@ -9,11 +9,12 @@
 
 {% set product_id = node.product_id|default(netweaver.product_id) %}
 {% set product_id = 'NW_DI:'~product_id if 'NW_DI' not in product_id else product_id %}
+{% set inifile = '/tmp/aas.inifile'~instance_name~'.params' %}
```

Comment:
maybe this variable can be name `aas_inifile`

Reply:
As we have the salt files split into each component already, I don't see the need to specialize the variable.

Reply:
I chose this implementation, because even though it's an strange scenario, we could have more than one `aas` or `pas` in the same host, with different `sid` and `instance number`. Just to make the file name more meaningful and descriptive. I think it's a good practice.

Reply:
ok